thirdeye-worker/src/main/java/ai/startree/thirdeye/worker/task/runner/DetectionPipelineTaskRunner.java

```diff
@@ -199,4 +206,19 @@ private void postExecution(final OperatorResult result) {
       }
     }
   }
+
+  private AnomalyDTO setAnomalyAuth(final AnomalyDTO anomaly) {
```

Comment:
Is there a better place for this method?

Reply:
Let's start here and refactor as needed. for now lgtm